### ISSUE TITLE
License Information

### ISSUE THREAD

**Issue by srcspider** [tony2001/pinba_engine#49]
Would it be possible to have Pinba available under BSD-style license? (http://www.linfo.org/bsdlicense.html)

Something like a dual-license with GNU GPL; if you have strong feelings about using GNU GPL.

-

If possible can we also have license information be more clearly mentioned in various places such as the README. A LICENSE file on the source root (and site) would also be appreciated since it doesn't really matter if you say it's license X if there's not a copy of license X I believe. Currently there's only a passing mention in the FAQ.

Since the license information is so non-existent, I feel I should probably mention this, just to share my experience with the licence. In general I find the GNU GPL license has essentially wide spread ban in most software shops over a wide variety of software disciplines unless it's a tool that comes directly bundled with the operating system or is available. The LGPL has somewhat of a better reputation, but is still quite not that well received.

Basically the issue is that GPL is that a viral license and software that integrates into other software that makes use of it might as well be considered part of the "GPL walled garden." If you're not a piece of software that has to be compiled into other software then this is not really a issue technically, but software houses just can't be bothered one bit about making the difference, since GPL's clauses essentially make any mistake suicidal. As such, more or less, anything whatsoever that uses GPL is much easier to just out right ban. The inaccessible legal babel and overly long length of it doesn't help either.

**Comment by fizerkhan** [comment]
+1
